Testing/PythonTests/PythonTests.cpp

```diff
@@ -89,7 +89,7 @@ TEST(pythonTests, extractLabelTest)
   run_use_case("extractlabel.py");
 }
 
-TEST(pythonTests, closeHolesTest)
+TEST(pythonTests, closeholesTest)
```

Comment:
What is the reasoning behind the inconsistent use of camel case for these tests?